source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ImageRenderer.java

```diff
@@ -52,70 +53,50 @@ public static ImageRenderer getInstance()
         return s_instance;
     }
 
-    private class ImageLoaderAsync extends AsyncTask<String, Void, HttpRequestResult<Bitmap>>
+    private class ImageRendererImageLoaderAsync extends ImageLoaderAsync
     {
-        ImageLoaderAsync(RenderedAdaptiveCard renderedCard, Context context, ImageView imageView, ImageStyle imageStyle)
+        ImageRendererImageLoaderAsync(RenderedAdaptiveCard renderedCard, ImageView imageView, ImageStyle imageStyle)
         {
-            m_context = context;
-            m_imageView = imageView;
+            super(renderedCard, imageView);
             m_imageStyle = imageStyle;
-            m_renderedCard = renderedCard;
         }
 
         @Override
         protected HttpRequestResult<Bitmap> doInBackground(String... args)
         {
-            try
-            {
-                Bitmap bitmap = null;
-                byte[] bytes = HttpRequestHelper.get(args[0]);
-                if (bytes == null)
-                {
-                    throw new IOException("Failed to retrieve content from " + args[0]);
-                }
-
-                bitmap = BitmapFactory.decodeByteArray(bytes, 0, bytes.length);
-                if (bitmap != null && m_imageStyle == ImageStyle.Person)
-                {
-                    Bitmap circleBitmap = Bitmap.createBitmap(bitmap.getWidth(), bitmap.getHeight(), Bitmap.Config.ARGB_8888);
-                    BitmapShader shader = new BitmapShader(bitmap,  Shader.TileMode.CLAMP, Shader.TileMode.CLAMP);
-                    Paint paint = new Paint();
-                    paint.setShader(shader);
-                    Canvas c = new Canvas(circleBitmap);
-                    c.drawCircle(bitmap.getWidth()/2, bitmap.getHeight()/2, bitmap.getWidth()/2, paint);
-                    bitmap = circleBitmap;
-                }
-
-                if (bitmap == null)
-                {
-                    throw new IOException("Failed to convert content to bitmap: " + new String(bytes));
-                }
-
-                return new HttpRequestResult<Bitmap>(bitmap);
-            }
-            catch (Exception excep)
-            {
-                return new HttpRequestResult<Bitmap>(excep);
-            }
+            return super.doInBackground(args);
         }
 
         @Override
         protected void onPostExecute(HttpRequestResult<Bitmap> result)
         {
-            if (result.isSuccessful())
-            {
-                m_imageView.setImageBitmap(result.getResult());
-            }
-            else
+            super.onPostExecute(result);
+        }
```

Comment:
do we need to override onPostExecute and doInBackground since all they do is call super?